Bottom: a5db68409f39b9565049fe8568e5102176bc1f52
Top:    5826532fd029f8768e721981114816ac780d2dfd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-07 00:06:08 +0100

TEMP TEST


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index ea77412..878cbf3 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -280,6 +280,7 @@ have_crosshair_attachments (void)
 static gboolean
 ghid_idle_cb (gpointer data)
 {
+  return;
   if (Settings.Mode == NO_MODE)
     SetMode (ARROW_MODE);
   ghid_mode_cursor (Settings.Mode);
